src/controllers/artist.js

```diff
@@ -0,0 +1,107 @@
+const db = require('../db/index')
```

Comment:
This file needs formatting as the indentations are 4 instead of 2 spaces and you have double quotations instead of single, which is what is in your prettier config file